src/Poseidon/CLI/Forge.hs

```diff
@@ -191,7 +191,7 @@ runForge (
                     P.tee outConsumer
             let startAcc = liftIO $ VUM.replicate (length newEigenstratIndEntries) 0
             P.foldM sumNonMissingSNPs startAcc return forgePipe
-        ) (\e -> throwIO $ PoseidonGenotypeExceptionForward e)
+        ) (throwIO . PoseidonGenotypeExceptionForward)
```

Comment:
Haha - ok - HLS already suggested this change to me, but I did not follow it, because I find the explicit lambda function more readable. Same for multiple other changes here :sweat_smile:

Reply:
Yeah, I didn't follow all HLS linter suggestions, but this one I liked. Definitely up to taste, and I'm happy to skip these in the future if you prefer the lambdas. Good to know.